interapp-backend/api/routes/endpoints/announcement/announcement.ts

```diff
@@ -41,7 +41,7 @@ announcementRouter.get(
   validateRequiredFields(AnnouncementIdFields),
   verifyJWT,
   async (req, res) => {
-    const query = req.query as unknown as z.infer<typeof AnnouncementIdFields>;
+    const query = res.locals.query as unknown as z.infer<typeof AnnouncementIdFields>;
```

Comment:
i forgot about this but ``as unknown as ...`` is no longer necessary since res.locals.query is ``any``
use ``const query: ... = res.locals.query`` instead to assert the type

Reply:
Done!